tool/actions-plan-preview/planpreview.go

```diff
@@ -178,11 +178,13 @@ func makeCommentBody(event *githubEvent, r *PlanPreviewResult) string {
 	b.WriteString(fmt.Sprintf(hasChangeTitleFormat, event.HeadCommit, len(r.Applications)))
 
 	changedApps, pipelineApps, quickSyncApps := groupApplicationResults(r.Applications)
+	if len(changedApps)+len(pipelineApps)+len(quickSyncApps) > 0 {
+		b.WriteString("\n## Plans\n\n")
```

Comment:
```suggestion
		b.WriteString("\n## Plans Preview\n\n")
```
how about this?

Reply:
umm, if u wanna do this, I feel `Plan Preview` or `Plan Previews` are better.

Reply:
Then I think I will go with `Plan Preview` but it's up to you 👍

Reply:
IMO, I feel `Plans` is enough, so please keep this. 🙌